prac_08/taxi_simulator.py

```diff
@@ -3,18 +3,36 @@
 from prac_08.taxi import Taxi
 
 MENU = "1) choose taxi\n2) Drive\n3) Bill\n4) Quit"
+
+
 def main():
     taxis = [Taxi("Prius", 100), Taxi("Kia", 150), SilverServiceTaxi("Hummer", 200, 4)
         ,SilverServiceTaxi("Limo", 250, 6)]
     total_bill = 0
-    print(MENU)
-    choice = int(input("your choice"))
+    choice = int(input("{}\n>>>".format(MENU)))
+    current_taxi = "none"
```

Comment:
In the question it mentions current_taxi = None
```suggestion
    current_taxi = None
```